frontend/js/utils/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const baseUrl = window.location.host;
+
+const apiUrls = {
+  battleDetail: (id) => `api/battle/${id}`,
+  currentUser: 'api/user/',
+};
```

Comment:
With `django-js-reverse` this is not required

Reply:
great!